packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -28,26 +33,67 @@ class AccountImportOptions extends Component {
       location: { pathname }
     } = this.props;
     const currentStep = pathname.slice(-1);
+
     history.push(`/accounts/new/${+currentStep + 1}`);
   };
 
   handlePhraseChange = ({ target: { value: phrase } }) => {
```

Comment:
Shouldn't check only the last word but also any previous word. i.e. if I type `cost fether ` it says "fether isn't a valid word", but if I type `cost fether field` it doesn't complain